native/common/jni-util.cpp

```diff
@@ -665,7 +665,7 @@ jboolean GetMapBoolValue(JNIEnv* env, jobject map, const char* optionName, jbool
 	}
 	return JNI_TRUE;
 }
-jboolean GetMapStringValue(JNIEnv* env, jobject map, const char* optionName, char* rv, unsigned long rvlen)
+jboolean GetMapStringValue(JNIEnv* env, jobject map, const char* optionName, char* rv, unsigned long rvlen, unsigned long* rvCount)

```

Comment:
I thought this was supposed to get changed in native/dll/DShowPlayer-2/PlaybackGraph.cpp#L278 in that last set of changes you did?  Did that somehow get dropped?  rvCount is unused in the only place that calls this function so we don't need to have it.